File: 100-Days-Series/Day 39/UnionOfTwoArrays.py

# Function to find unique elements.
def getUnionOfList(arr1,arr2):
    uniArray=[]
    k=0
    for i in range(0,len(arr1),1):
        uniArray.append(arr1[i])
        k+=1
    for i in range(0,len(arr2),1):
        isPresent=False
        for j in range(0,k,1):
            if arr2[i]==uniArray[j]:
                isPresent=True
                break
        if not isPresent:
            uniArray.append(arr2[i])
            k+=1
    return uniArray

# Let's Start here
arr1=[1,2,3,4,5,6,7,8]
arr2=[5,6,7,8,9,10,10]
print("Union of Arrays : ")
print(getUnionOfList(arr1,arr2))

File: 100-Days-Series/Day 39/UnionOfTwoArrays.cpp
#include<iostream>
#include<vector>
using namespace std;
// Function  to find unique elements
void getUnion(const vector<int> arr1,const vector<int> arr2){
    int uniArray[arr1.size()+arr2.size()];
    int k=0;
    for(int i=0;i<arr1.size();i++){
        uniArray[k++]=arr1[i];
    }
    for(int i=0;i<arr2.size();i++){
        bool isPresent=false;
        for(int j=0;j<k;j++){
            if(arr2[i]==uniArray[j]){
                isPresent=true;
                break;
            }
        }
        if(!isPresent){
            uniArray[k++]=arr2[i];
        }
    }
    cout<<"\nUnion of arrays: "<<endl;
    for(int i=0;i<k;i++){
        printf("%d ",uniArray[i]);
    }

}
// Let's Start here
int main(){
    vector<int>arr1={1,2,3,4,5,6,7,8};
    vector<int>arr2={5,6,7,8,9,10};
    getUnion(arr1,arr2);
}